src/main/java/at/hannibal2/skyhanni/features/mining/OreBlock.kt

```diff
@@ -86,7 +86,7 @@ enum class OreBlock(
     JADE(EnumDyeColor.LIME, { inCrystalHollows || inGlacite }),
     SAPPHIRE(EnumDyeColor.LIGHT_BLUE, { inCrystalHollows || inGlacite }),
     TOPAZ(EnumDyeColor.YELLOW, { inCrystalHollows || inGlacite }),
-    JASPER(EnumDyeColor.MAGENTA, { inCrystalHollows || inGlacite }),
+    JASPER(EnumDyeColor.MAGENTA, { inCrystalHollows || inGlacite || inMineshaft }),
```

Comment:
`inGlacite` already includes mineshafts, this seems unnecessary.

Reply:
> inGlacite already includes mineshafts, this seems unnecessary.
didnt work for me today. So I'm not sure about that

> Also, there are no commissions to mine Jasper gemstones.
you are right, there aren't. but there are still jasper gemstones, and they aren't grayed out properly.

Reply:
> Ah, does it gray out gemstones that are not for commissions?

this is the whole point of the feature. otherwise, having the correct blocks highlihgt in any color will just cause confusion with the natural color of other blocks

> I can't see from the code how it could possibly fail inGlacite but not inMineshaft

after looking into the code a bit more, i also cant see how this should fix anything. will investigate further when i notice it the next time